main.py

```diff
@@ -0,0 +1,48 @@
+import random
+
+names = []
+
+
+def add_name():
+    print('Press x stop entering a name')
```

Comment:
Use consistent quotation marks: sometimes you use single quotes `''`, and other times you use double `""`.